src/components/templates/ProductTemplate/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { ProductTemplateLoading } from './ProductTemplateLoading/ProductTemplateLoading';
+import { ProductTemplateProgress } from './ProductTemplateProgress/ProductTemplateProgress';
+import { ProductTemplateCard } from './ProductTemplateCard/ProductTemplateCard';
+import ProductTemplateComponent, { ProductTemplateProps } from './ProductTemplate/ProductTemplate';
+import React from 'react';
```

Comment:
Think the standard in zrc is to have the package imports at the top - followed by the relative imports with a space separator

Reply:
We can add https://eslint.org/docs/2.0.0/rules/sort-imports in the future. To force that